src/tasks/withdraw/settle.ts

```diff
@@ -27,15 +27,8 @@ export async function submitSettlement({
   gasEstimator,
   encodedSettlement,
 }: SubmitSettlementInput) {
-  if (!isSigner(solver)) {
-    const settlementData = settlementContract.interface.encodeFunctionData(
-      "settle",
-      encodedSettlement,
-    );
-    console.log("Settlement transaction:");
-    console.log(`to:   ${settlementContract.address}`);
-    console.log(`data: ${settlementData}`);
-  } else if (
+  if (
+    isSigner(solver) &&
```

Comment:
This changes the behaviour of other scripts that use the `submitSettlement` method.